hackathon_site/event/api_views.py

```diff
@@ -5,14 +5,13 @@
 from rest_framework.exceptions import ValidationError, PermissionDenied
 from rest_framework.response import Response
 
```

Comment:
this file shouldn't have changes, could you revert this file to what's on develop